libs/common/include/launchdarkly/attributes_builder.hpp

```diff
@@ -155,17 +180,39 @@ class AttributesBuilder final {
         return *this;
     }
 
+    /**
+     * Start adding a kind to the context.
```

Comment:
Wording nit: I think you are adding a new context with the specified kind to the builder ?

Reply:
(The kind happens to be a context, like the address happens to be a string, or the name is a string.)